src/app/beer_garden/api/entry_point.py

```diff
@@ -92,14 +97,15 @@ def start(self) -> None:
                 self._signal_handler,
                 self._ep_conn,
                 lpm.lpm_proxy,
+                self.ep_config,
             ),
         )
         self._process.start()
 
         # And listen for events coming from it
         self._event_listener.start()
 
-    def stop(self, timeout: int = None) -> None:
+    def stop(self, timeout: int = None, closeCommunicationPipes=False) -> None:
```

Comment:
Please don't use CamelCase for things. See https://stackoverflow.com/a/159745 (specifically https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)